### PR TITLE
Master port rhel 1290858

### PR DESCRIPTION
**Pull request by rvykydal**
This is just a backport from rhel7-branch to master, creating PR to run CI tests.

### CODE REVIEW

File: pyanaconda/ui/gui/spokes/custom.py [hunk 1]
```diff
@@ -2291,7 +2292,8 @@ def on_container_changed(self, combo):
         user_changed_container = True
         if create_new_container:
             # run the vg editor dialog with a default name and disk set
-            hostname = self.data.network.hostname
+            if hostname == network.DEFAULT_HOSTNAME:
```
Comment (bcl) [inline pyanaconda/ui/gui/spokes/custom.py]: hostname hasn't been assigned yet.

File: pyanaconda/ui/gui/spokes/custom.py [hunk 2]
```diff
@@ -2596,7 +2598,8 @@ def _populate_container(self, device=None):
                 self._containerCombo.set_active(containers.index(c))
 
         if default_container_name is None:
-            hostname = self.data.network.hostname
+            if hostname == network.DEFAULT_HOSTNAME:
```
Comment (bcl) [inline pyanaconda/ui/gui/spokes/custom.py]: Same here.

**Comment by rvykydal** [comment]
Thank you for the catch bcl and pylint tests, I've screwed manual port of the patch.

**Comment by bcl** [comment]
Looks good now :)

**Comment by rvykydal** [comment]
One more CI round to fix glade translation context check fail.

**Comment by rvykydal** [comment]
Pushed

**Pull request closed without merging**